### ISSUE TITLE
Remove The Spare

### ISSUE THREAD

**Issue by nuklearcellphoneg** [d3athrow/vgstation13#24788]
[controversial]

The spare exists only because the situation can arise that 

1. There is no Captain or HoP

2. There are no admins on or a way to contact the admins.

As admins are essentially now in constant contact via ahelps being relayed to the discord, the benefits per round far outweigh keeping it in. If the crew is capable of breaking into the Captain's office, they are more than capable of breaking to Engineering or Medbay to set up power or revive players, the two necessities to an ongoing round.

**Comment by ghost** [comment]
If there's no spare then how am I supposed to rush it? Checkmate, atheists.

**Comment by nuklearcellphoneg** [comment]
When you get down to it, the sole reason the spare exists is to be able to call the shuttle, if there are no admins online and no Cap or HoP. This really isn't the case any longer, and if you want to push it even further, you can code in a function to request a shuttle call, or that if a non head requests the shuttle, it sends a message to admins in a sekrit shuttle call channel... or hell, make it where anyone can call the shuttle, but there's a verification timer, or some such nonsense.

**Comment by FudgePucker** [comment]
I still like my idea of requiring every head stamp around a crayon rune in order to get the spare.

**Comment by boy2mantwicethefam** [comment]
wtf if there is no spare how can i remain angry at spare rushers

**Comment by juanototo2** [comment]
Epically downboated
If you honestly think this PR will go through you are brain fucking damaged

**Comment by nuklearcellphoneg** [comment]
I'm not. I'm not sure you know what a PR is, first, and second, can you give me one good reason not to remove the spare? @juanototo2

**Comment by nuklearcellphoneg** [comment]
please respond

**Comment by juanototo2** [comment]
Doing a kneejerk reaction, like removing the spare, causes things built around its existence; like, say, deadpop, for example, is bad because, once the action is completed, you either revert the action or have to fix everything that was broken by merging it. You'd have to add a Call Shuttle vote; you'd have to add a way to deal with nocaptainnoHoP in deadpop without the spare, etcetera.
Besides, removing the spare removes one of the things that makes space, well, space; if you're mad about non-authorized hands touching it, they're VALID. You have GUNS and WEAPONS. THEY CAN'T DEFEND THEMSELVES WITHOUT GETTING BANT. ANY CHEM DISPENSER CAN CREATE A ITEM THAT CAN DELETE BRAINS. CONNECT THE FUCKING DOTS.

**Comment by juanototo2** [comment]
Also, overwhelming negativity on a PR generally makes that PR unmergeable, and the inverse is true; see the emagged nikita PR.

**Comment by juanototo2** [comment]
It really makes you think when a single nonantagonist without any tools or ranged stuns to speak of and no license to kill in any way, shape or form causes security mains to be more mad than actual antagonists doing the epic 12:06 singuloose dance or silent depop dance.

**Comment by nuklearcellphoneg** [comment]
1. This is not a PR

2. You simply send a fax to Nanotrasen(the admins) that the shuttle needs to be called, or allow(code)s anyone to call it, or allow the highest ranking member of a department to call it.

3. Why are you calling this a kneejerk reaction? I've played this game, on this specific server, since 2013. Adding the spare into the game was originally a knee jerk reaction due to if a Head was not on, and no Admins were on, there was no way to get the shuttle called- that is no longer true.

4.  'You have GUNS and WEAPONS. THEY CAN'T DEFEND THEMSELVES WITHOUT GETTING BANT. ANY CHEM DISPENSER CAN CREATE A ITEM THAT CAN DELETE BRAINS. CONNECT THE FUCKING DOTS.'  lol.... what? I don't think antags or HoSes or assistants should have it. 

5.  It really makes you think when a single nonantagonist without any tools or ranged stuns to speak of and no license to kill in any way, shape or form causes security mains to be more mad than actual antagonists doing the epic 12:06 singuloose dance or silent depop dance.


oh man I'm sorry you've got me laughing too hard now I can't maintain a semblance of seriousness at this point. I play Bob Dylan, Solid Snake, and I random name. Every round I play in this game, I break into departments and steal stuff and get thrown in the brig and attempt to yakkety sax, break out, yadda yadda.

**Comment by nuklearcellphoneg** [comment]
OK let me take a step back.

Removing the spare makes a lot of things more interesting, both when there is and is not a Cap or HoP on board. If there is no HoP or Cap, then someone breaks into the Cap's office, they steal the spare, they get all access, they can now go anywhere.

If there is no spare to be stolen, this means that every time they wish to go into a place they have no access to,  they must break in.

1. While breaking in, they are at risk of getting caught in the act.

2. They risk leaving behind evidence of the break in.

3. Areas such as the AI core are much more difficult to break into because of this.

And so on.

**Comment by nuklearcellphoneg** [comment]
I believe the spare has cradled this generation of players into being soft and unrobust unfortunately. For shame utilitas you should crave hardcores

**Comment by juanototo2** [comment]
1. Okay then. Remind me again when you eventually code it and it gets downboated into oblivion.
2. Fax to Nanotrasen (TM) Okay, sure. Will there be any restrictions on the fax? Can it be wrenched and disposed of in the conventional manner? allows anyone to call it: You're still breaking into the bridge, by the way. How would you deal with the conventional issues involved in this, like Shitter McGee (Assistant/Scientist/Mechanic) test-wiring into the bridge, shuttlerushing because he didn't get antag today and all of the antags are dead, and then wrenching the computer and dragging into Falsewalled Maint Cubbyhole #189137?
3. It's a kneejerk solution because, by definition, it is. It removes a feature that's been in the game for a long time, with previous PRs that have already attempted to alleviate its effects on the round, AND administration already starting to act upon the feature's effects on the round, with shit like spare bears, that one spare cereal event, etcetera. The problem with removing old features is that everything else was balanced around it. I already know you're going to refute this with something something telescience, but Telescience isn't removed as a feature completely from roundstart; paramedics have it, Science can have it with willful assistance by the RD, etcetera, and important areas are teleblocked because of the cancer that not having teleblock was. Unlike this that you're planning on doing.
4. So you're saying that neither a traitor nor an assistant nor Femmestatic HoS #198318 should have the spare? Curious, because I always thought that most anti-spare people were fine as long as command or security had the spare; because they could show up to the HoP line with it for access changes, they didn't give out all access like it's candy. Again. Grand trespass into the captain's office or armory is evidence of acting like an antagonist, allowing them to be field executed by any security or command member happening to be in the area at the time. 
5. No response.
I don't ALWAYS play assistant, and neither do I play ablino on the regular anymore. I've stopped doing that now because thanks to the dynamic changes it isn't cultopsblobwiz 24/7 every single round, so I don't need all of the tools fucking constantly. Neither do I ALWAYS rush the spare forever as an assistant. If I were a unrepentant asshole, I would've been perma'ed a long time ago, but I haven't ever unironically silently depopulated as an antagonist, so I rest my case.

Post 2:
1. You're implying that people use the spare as a device for entry into non-department rooms on the regular. Once the colloquial arrivals ID computer gets used, for the grand majority of people who use it they don't immediately rush to the armory and then to telecomms and the upload to upload some godawful piece of crap to both. No, the grand reason why most people use the spare is for opening lockers, lockboxes and other restricted items; the main reason to gain access to a non-department room is a crisis necessitating access, e.g. break into viro because Deity decided that a comfy job should be forced to sit through 6 ebola outbreaks every single round, an item that you need or want, e.g. assistant breaking into engi lobby on Box for the toolbelts, or using the department tools, machinery and items to antagonize; e.g. 12:06 blasmafludd, 12:34 baguloose, 12:12 shardbomb. etcetera. If you want to get Engineering access or atmos access to be a giga-autist, then you can ask the HoP because it's not of such urgency you need to break in, and you're gonna be sitting in there for long periods of uninterrupted time.
I would continue the rest but my browser bugged a bit and you're still waiting for a response. Besides, real life will call in a bit.

**Comment by juanototo2** [comment]
>risk of getting caught in the act

Are you joking me, man? Do you know how easy it is for people to screwdrive, pulse test, crowbar, screw again, repeat? Besides, WHO, apart from a brigvalids-searching AI is going to look at said engineering lobby and tech storage multitool/glubbs every single round like a hawk?

>evidence

Hahahahhahahahahahhahahahhaahaha
What was the last time the Detective ever was useful? Either he's hunting a target of too little importance, or the target already waltzed face first into the nearest populated area and started antagonizing or blasmafludded or shot your security officer in the face because he's flarg or Absolutely. The current state of the rounds do not allow the detective to be useful; at most, he'll be one free catch and confirm for a shittier security officer. 

>areas such as the AI core

What is like three EMPs or going into the rwalls at the back or railguns or gaming the turrets with girders or literally just asking your way in for 500$, Nuklear?

**Comment by jknpj** [comment]
Just make deadpop start on egalitarian mode

**Comment by utilitas-** [comment]
\>implying I need the spare to set up the engine, set cryo and do everything I need (except for robotics because fuck you apparently)
But have fun closing all the hacked open doors I leave behind, bucko. Not having the spare only wastes time.
And you are wasting everyone's time by trying to get it removed or changed in any way whatsoever.

**Comment by nuklearcellphoneg** [comment]
Okay, you have spit out a wall of text. Reduce your wall of text please, because neither I nor you brought up that many points to begin with. Let me tackle the few that have the merit of being poured over.

1. Calm down there sport, you're going on like my little cousin when he got into his brother's ADD meds.

2. You can already fax nanotrasen, go for it. You do whatever it is you do currently when greyshit mcshitter steals the spare and calls it because he didn't get antag. Only difference is, now he's breaking into the bridge instead of the Cap office.

3. It's not a feature, it's an ID card. I have no clue what you're going on about with telescience there, so I'll move on.

4. I would like to remove it because it makes the game much less interesting, there are already mechanics in the game which bypass the need for access, for example hacking doors, bossing the AI around, blowing open locked crates, and soldering the cargo's innards so that anyone can order from it- mimic this with robotics, the shuttle, RND and the Brig timer and we're golden, no?

5. In the past, I pushed stacking reagents for healing as a meta early on, I am part of the filling up autoinjectors with chloral meme(God rest their soul), I have gone through the being praised as a gr8 medbay player meme team, I have gone through the constantly shitposted for being a massive powergamer meme- I have a lot of rounds clocked in this game, and specifically this server. I am very invested, and if I push for something, it is not for my personal interest or agenda. I get very bored very easily with the game now. The last time I spent a good month or so autisming, I noticed we had at some point added bees, and after a couple of weeks, I crashed a lot of peoples' clients. If I push for a mechanic, or a tweak, or a balance, it is by no means in my own personal interest or agenda, but because I see a lot of potential in what that would bring. With no Captain or HoP presently, in order to open a Bee crate with no Botanist or Cargo, you would: Seek out the spare, order the crate, open it. If there was no spare you would: Go into cargo, take apart the console, solder its innards, put it back together, order the crate, select your poison in regards to how you blow open the crate, blow open your crate, hack into Botany, set up your bees. The spare invites singular and limited interactions within the, what do the magazines call it, 'emergent gameplay', whereas having no spare compels the crew folk to cooperate or otherwise innovate in order to overcome obstacles. You may replay that this adds tedium, but this is a game with a Janitor, for God's sake, its foundation is tedium.

Post 2. lmao why are you going into virology to cure it(also just break in)

>risk of getting caught in the act

Why yes- there is more risk getting caught in hacking open a door, or opening up a window, than in simply opening and shutting a door, you cannot argue there is not more risk regardless of its marginal nature.

>evidence

Why yes- there is more evidence of all varieties created when you don't have all access. For example, if you open a locked crate, there is the evidence of whatever method by which you opened said crate.

>areas such as the AI core

Why yes- using EMPs, thermite, so on and so forth, is much more noticeable that simply opening and shutting a door via all access.

>But have fun closing all the hacked open doors I leave behind, bucko. Not having the spare only wastes time.

And hey, I'm fine with that- I would prefer you leave the doors open in case I need something out of there, after all, nobody has access.

**Comment by utilitas-** [comment]
Either everybody has access or nobody has access. Do you not _see_ how the result is _exactly the same_? Do you not see that _it isn't the fucking spare, it's the individuals who use the spare that are at fault?_
Fucking hell, Nuke. And that's not even bringing up the fact that - hey, maybe _someone else_ would like to use our codebase for their own server now that TG is absolute fœtid dogtwaddle. We are not coding purely for our server, with our issues and our players. We're coding for everyone that might want to use these features.
Out-arguing everyone else to the point of exhaustion does not automatically make you correct by default, Nuke. Nobody's right. I'm outski.

**Comment by nuklearcellphoneg** [comment]
Chill my dude, I even made it a point to keep all my stuff fairly brief- the only thing that's long winded here is my replying to that dude there my man.

**Comment by Durgen** [comment]
sending a fax requires bridge access, which it is possible no one could have

**Comment by incakola** [comment]
when the command report prints out, let it print also out a spare if no heads with the ability to change access are on
or if the command report gets faxed with head stamp(s), then centcom faxes back a spare
maybe the spare it sends back has limited uses or only HoP access

**Comment by SGuivenne** [comment]
I'm just going to say it.  The idea that we don't need the spare ID because an admin SHOULD be able to handle it via admin PM is one of the dumbest things I've ever heard.  Its easily possible to have no admins during deadpop, when the spare is most needed, and even then, do you really want to have to be constantly pinged in discord to drop shit you're doing and jump into ss13 because "reee gibe spare"?

**Comment by Kurfursten** [comment]
I liked the idea I heard that the spare should only appear in lowpop. Above 20 crew or so just don't spawn.

**Comment by utilitas-** [comment]
what if the crew has no HoP, no Captain, no engineers and everyone is screaming at no one in particular to set up the power?
same exact result: a lot of breakins and crime because some shitter coder tried to fix what ain't broke and never was broke.
if you see some massive faggot repeatedly rushing the spare, what do you do? here's a hint for all y'all clever folk out there: what do you do with people who break them rurus?
hint hint hint: _you ban'em_.
and if the admins are shit?
hint hint hint: _play the game and cunt punt the staticnaming shitter that's in the captain's quarters humping the antique case three minutes in_
_if he retaliates, you have every opportunity to go cry on the forums_

**Comment by FudgePucker** [comment]
I played a round recently where I was the captain after setting it to medium priority, I had to get my cats some water, sixty seconds until round starts, I should have enough time. Nope. Came back to three assistants in my office before 12:05. Spent the entire rest of the round hunting them down and then they brag about how shit I was and how I was a condom for not "securing" the spare and disk.

Fuck spare rushing.
Remove the spare if there is a HoP or Captain.
